AndroidApps/BundleTransport/app/src/main/java/net/discdd/bundletransport/viewmodels/TransportUsbViewModel.kt

```diff
@@ -92,7 +93,10 @@ class TransportUsbViewModel(
                     return@withContext
                 }
                 val destinationDir = createIfDoesNotExist(currentUsbDir, "toClient")
-                val bundlesToTransfer: List<File> = sourceDir.walk().filter(File::isFile).toList()
+                val bundlesToTransfer: MutableList<File> = sourceDir.walk().filter(File::isFile).toList().toMutableList()
+                //val bundlesToTransfer: MutableList<File> = sourceDir.walk().filter(File::isFile).toList() as MutableList<File>
+                bundlesToTransfer.add(mailApkFile)
```

Comment:
i think this will put it into the toClient directory. we probably want it right in the ddd_transport directory.